test/client.go

```diff
@@ -12,6 +14,13 @@ import (
 )
 
 func main() {
+	if err := run(); err != nil {
```

Comment:
Why do we need to change this to add lint to CI?

Reply:
Linter was complaining to not use log.Fatal in a non-main func